modules/cluster-and-ha-management/pages/region-aware.adoc

```diff
@@ -0,0 +1,198 @@
+
+This is an example `install.conf` that enables region-awareness for non-interactive installation.
+
+[#_install_conf_example]
+[,javascript]
+----
+{
+  "BasicConfig": {
+    "TigerGraph": {
+      "Username": "mark",
+      "Password": "markpassword",
+      "SSHPort": 22,
+      "PrivateKeyFile": "",
+      "PublicKeyFile": ""
+    },
+    "RootDir": {
+      "AppRoot": "/home/tigergraph/tigergraph/app",
+      "DataRoot": "/home/tigergraph/tigergraph/data",
+      "LogRoot": "/home/tigergraph/tigergraph/log",
+      "TempRoot": "/home/tigergraph/tigergraph/tmp"
+    },
+    "License": "<license>",
+    "RegionAware" : true,
+    "NodeList": [
+      "us-east-1: m1: 123.456.78.99",
+      "us-east-2: m2: 123.456.78.98",
+      "us-east-3: m3: 123.456.78.97",
+      "us-east-3: m4: 123.456.78.96"
+    ]
+  },
+  "AdvancedConfig": {
+    "ClusterConfig": {
+      "LoginConfig": {
+        "SudoUser": "tom",
+        "Method": "K",
+        "P": "<sudo_user_password>",
+        "K": "/home/tom/mykey.pem"
+      },
+      "ReplicationFactor": 2
+    }
+  }
+}
+----
+
+The configuration items that need to meet the conditions are
+
+* **RegionAware** should be `true`
+* **NodeList**
+** The number of nodes is more than 3
+** The format of all nodes is `Region:HostID:IP/HostName`
+* **ReplicationFactor** is greater than 1
+
+
+=== After installation
+
+To enable region-awareness in an existing cluster, you can use xref:expand-a-cluster.adoc[Cluster Expand].
+
+[TIP]
+====
+You can check whether the node information meets the requirements according to the output of the following command:
+[source, console]
+----
+$ gadmin config get System.HostList
+----
+
+Legal::
+[source,javascript]
+----
+[{"Hostname":"192.168.0.1","ID":"m1","Region":"us-east-1"}, {"Hostname":"192.168.0.2","ID":"m2","Region":"us-east-2"}, {"Hostname":"192.168.0.3","ID":"m3","Region":"us-east-3"}]
+----
+
+Illegal::
+[source,javascript]
+----
+[{"Hostname":"192.168.0.1","ID":"m1","Region":""}, {"Hostname":"192.168.0.2","ID":"m2","Region":""}, {"Hostname":"192.168.0.3","ID":"m3","Region":""}]
+----
+If any host does not contain region information, use the following command to add it:
+
+[source,console]
+----
+$ gadmin config entry System.HostList
+$ gadmin config apply -y
+----
+
+====
+
+If you just want to enable region-aware without any other change, use the command following:
+
+[source,console]
+----
+$ gadmin cluster expand --region-aware
+----
+
+If the existing cluster does not meet the prerequisites, you can also add nodes and modify replication factor with one command, example:
+
+[source,console]
+----
+$ gadmin cluster expand us-east-3:m3:192.168.0.3,us-east-4:m4:192.168.0.4 --ha 2 --region-aware
+----
+
+== Disable Region Aware
+
+Disable region-awareness via xref:expand-a-cluster.adoc[Cluster Expand]
+
+[source,console]
+----
+$ gadmin cluster expand --region-aware=false
```

Comment:
Does this work? And, do we really the cluster expansion in this case? Or it is a no-op?

Reply:
Yes, we do expand process to achieve region-aware